app/models/transaction.rb

```diff
@@ -3,6 +3,8 @@ class Transaction < ApplicationRecord
   
   scope :success, -> { where result: 'success' }
 
+   scope :success, -> { where result: 'success' }
```

Comment:
Whoops! Thanks, @case-eee. Sometimes I trust git to figure things out a bit too much.